tests/bundle/Templating/Twig/Extension/YoutubeIdExtractorExtensionTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+namespace EzSystems\Tests\EzPlatformRichTextBundle\Templating\Twig\Extension;
+use EzSystems\EzPlatformRichTextBundle\Templating\Twig\Extension\YoutubeIdExtractorExtension;
+use PHPUnit\Framework\TestCase;
+use Twig_Function;
+class YoutubeIdExtractorExtensionTest extends TestCase
+{
+    public function youtubeUrls()
```

Comment:
```suggestion
    public function getYouTubeUrls(): array
```
(... or `providerForTest...`)